x-pack/plugins/security_solution_serverless/server/cloud_security/defend_for_containers_metering.ts

```diff
@@ -76,7 +76,8 @@ export const getUsageRecords = async (
             {
               range: {
                 'event.ingested': {
-                  gt: searchFrom.toISOString(),
+                  // gt: searchFrom.toISOString(),
```

Comment:
I'd rather add the tech debt ticket to the comment, than leaving the old code commented out

Reply:
I solved it in https://github.com/elastic/kibana/pull/186777, so I remove the comment, and will merge the other PR first.